@stellar/design-system/webpack.config.js

```diff
@@ -9,11 +9,21 @@ module.exports = {
   output: {
     filename: "[name].js",
     path: path.resolve(__dirname, "build"),
-    libraryTarget: "commonjs-module",
+    libraryTarget: "umd",
```

Comment:
Useful info on why this should be "umd":

https://stackoverflow.com/a/55659242